docs/jwst/pipeline/main.rst

```diff
@@ -229,15 +204,94 @@ Control (WFS&C) use a dedicated processing flow through the pipeline stages.
  - Stage 1: WFS&C exposures use the same :ref:`calwebb_detector1 <calwebb_detector1>`
    pipeline processing and steps as regular images.
 
- - Stage 2: The ASN generator identifies WFS&C exposures as those with the
-   string "WFSC" within their "VISITYPE" keyword value. The resulting ASN
-   uses the :ref:`calwebb_wfs-image2 <calwebb_wfs-image2>` pipeline for stage 2
-   processing.
-   This pipeline configuration is identical to :ref:`calwebb_image2 <calwebb_image2>`
-   with the omission of the :ref:`resample <resample_step>` step.
+ - Stage 2: WFS&C exposures use the same :ref:`calwebb_image2 <calwebb_image2>`
+   pipeline processing and steps as regular images. CRDS reftype
+   ``pars-image2pipeline`` has a specific step parameter reference for WFS&C
+   processing. The processing is identical with other image processing except
+   for the omission of the :ref:`resample <resample_step>` step.
 
  - Stage 3: The ASN generator identifies pairs of dithered WFS&C images to be
    combined via the "PATTTYPE" keyword value "WFSC". The resulting ASN
    uses the :ref:`calwebb_wfs-image3 <calwebb_wfs-image3>` pipeline
    for stage 3 processing. This pipeline consists of the single step
    :ref:`wfs_combine <wfs_combine_step>`.
+
+Configuration File Deprecation
+==============================
+
+Up to version 1.1.1, the primary way specific pipelines were referred to was by
```

Comment:
We're only at 1.1.0 right now, so did you mean that version? I doubt we'll ever have a 1.1.1, so the next potential release would be 1.2.0 (probably an interim release before B7.8) and then B7.8 would be higher yet.